.github/workflows/export_standards_data.yml

```diff
@@ -42,56 +45,81 @@ jobs:
 
         sdk_version = ''
         if m_major:
-            sdk_version += m_major.groups()[0]
-            print("::set-env name=OS_SDK_VERSION_MAJOR::{}".format(m_major.groups()[0]))
+            OS_SDK_VERSION_MAJOR = m_major.groups()[0]
```

Comment:
I hesitate to say stuff like this because I don't want you to go off and nerd-snipe yourself into doing it, but downloading the OS SDK seems like it would be a good thing to factor into a separate action

Reply:
Yeah, we'll deal with that another day.